portugues/colaborando.md

```diff
@@ -5,6 +5,8 @@ Você:
 
 [Procura o repositório do aplicativo?](colabora/colaborar.md)
 
+[Começa a pensar como solucionar o bug e resolve olhar no codigo fonte](colabora/codFonte.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.